tests/test_evaluation/test_metrics/test_multi_metrics.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+from unittest import TestCase
+
+import numpy as np
+import torch
+
+from mmcls.evaluation.metrics import Accuracy
```

Comment:
The metric you are testing should not be 'Accuracy' it should be 
'MultiTaskMetric', no ?

```suggestion
from mmcls.evaluation.metrics import MultiTaskMetric
```